controller_interface/include/semantic_components/imu_sensor.hpp

```diff
@@ -0,0 +1,139 @@
+// Copyright 2021 PAL Robotics SL.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef SEMANTIC_COMPONENTS__IMU_SENSOR_HPP_
+#define SEMANTIC_COMPONENTS__IMU_SENSOR_HPP_
+
+#include <limits>
+#include <string>
+#include <vector>
+
+#include "hardware_interface/loaned_state_interface.hpp"
+#include "semantic_components/semantic_component_interface.hpp"
+#include "sensor_msgs/msg/imu.hpp"
```

Comment:
Thy dependencies need to be updateth

Reply:
Is this https://github.com/ros-controls/ros2_control/pull/429/commits/b8ff6c6b7839f7cc99ce26693bd6a7cf33fdb804#diff-d3a44eabaa35ebf41dcc01ca13650700fbc1dda7dfa18abf3defc3e465a6f85eL22 that evil? If not please make me worthy of the secrets to fight such evil!

Reply:
nvm, package.xml dependencies. Fixed

Reply:
you are worthy now!